plugins/woocommerce/tests/php/includes/gateways/paypal/class-wc-gateway-paypal-request-test.php

```
@@ -0,0 +1,201 @@
+		$order->set_total( 40 );
+		$order->set_currency( 'USD' );
+		$order->save();
+
+		$item = current( $order->get_items() );
+		$item->set_total_tax( 10 );
+		$item->save();
+
+		add_filter( 'pre_http_request', array( $this, 'check_create_paypal_order_params' ), 10, 2 );
+
+		$request = new WC_Gateway_Paypal_Request( new WC_Gateway_Paypal() );
+		$result  = $request->create_paypal_order( $order );
```

PHPMD (2.15.0)

Avoid unused local variables such as '$result'. (Unused Code Rules)

(UnusedLocalVariable)